packages/flutter_tools/test/general.shard/dart/generate_synthetic_packages_test.dart

```diff
@@ -262,3 +262,5 @@ void main() {
     );
   });
 }
+
+class FakeStackTrace extends Fake implements StackTrace {}
```

Comment:
You could instead construct a real stack trace via `StackTrace.current`, if you're not using it for any expectations

Reply:
I think the `Fake` is better since it enforces it's not used, and there's no overhead for creating a real `StackTrace`.  Up to you though, I can swap it if you prefer.

Reply:
I would prefer to use the real object if possible

Reply:
Done.